site.yml

```diff
@@ -45,6 +45,14 @@
       tags: ['infra', 'rabbitmq']
   environment: "{{ env_vars|default({}) }}"
 
+- name: setup serverspec
+  hosts: all
+  roles:
+    - role: serverspec
+      tags: ['serverspec']
+      when: serverspec.enabled|bool
+  environment: "{{ env_vars|default({}) }}"
```

Comment:
There was a reason that is escaping me at the moment. Let me go back through my old logs and I'll let you know.